activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -219,26 +219,28 @@ def write_serialized_entry(key, payload, **options)
         def read_multi_entries(names, **options)
           keys_to_names = names.index_by { |name| normalize_key(name, options) }
 
-          raw_values = begin
-            @data.with { |c| c.get_multi(keys_to_names.keys) }
-          rescue Dalli::UnmarshalError
-            {}
-          end
+          rescue_error_with({}) do
+            raw_values = begin
+              @data.with { |c| c.get_multi(keys_to_names.keys) }
+            rescue Dalli::UnmarshalError
```

Comment:
Can the `UnmarshalError` rescue be removed since it should be caught by the outer block rescuing `Dalli::DalliError`?

Reply:
Thanks sorry for the failing tests, I'm having an issue with my local setup and thought they actually passed.

Yeah I can remove that, since we're now rescuing a superclass, good catch!